app/views/dump.py

```diff
@@ -28,8 +28,8 @@ def dump_submission():
     answer_store = get_answer_store(current_user)
     metadata = get_metadata(current_user)
     session_data = get_session_store().session_data
-    schema = load_schema_from_session_data(session_data)
+    g.schema = load_schema_from_session_data(session_data)
```

Comment:
We should make this a separate PR as its unrelated to the change

Reply:
Agree, changing it